PyRDF/CallableGenerator.py

```diff
@@ -2,18 +2,32 @@
 
 class CallableGenerator(object):
     """
-    Class that generates
-    mapper and reducer functions
+    Class that generates a callable
+    to parse a PyRDF graph.
+
+    Attributes
+    ----------
+    root_node
```

Comment:
We might as well call the parameter `head_node`

Reply:
@etejedor , can I include this along with the PR to remove `_dfs` ?

That PR will be completely for cleaning up and refactoring variable names. So I guess it'd be better to include this in that PR. Is that okay ?

Reply:
Sure